<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MismatchWindow</class>
 <widget class="QDialog" name="MismatchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1585</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MismatchWindow</string>
  </property>
  <widget class="QLabel" name="expectedImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>720</width>
     <height>362</height>
    </rect>
   </property>
   <property name="text">
    <string>expected image</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultImage">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>170</y>
     <width>720</width>
     <height>362</height>
    </rect>
   </property>
   <property name="text">
    <string>result image</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultFilename">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>90</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>result image filename</string>
   </property>
  </widget>
  <widget class="QLabel" name="expectedFilename">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>700</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>expected image filename</string>
   </property>
  </widget>
  <widget class="QLabel" name="imagePath">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>1200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>image path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="passTestButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pass</string>
   </property>
  </widget>
  <widget class="QPushButton" name="failTestButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>600</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Fail</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abortTestsButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>600</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort Tests</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>600</y>
     <width>720</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>similarity</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
